### ISSUE TITLE
Need for Speed Unbound won't start after recent Vol. 7 Update

### ISSUE THREAD

**Issue by LoVega1337** [p0358/Fuck_off_EA_App#55]
### Checklist

- [X] I ensured I'm running the [latest version of the patch](https://github.com/p0358/Fuck_off_EA_App/releases), and that the issue is present in it if applicable
- [X] I have read the entirety of the [FAQ](https://github.com/p0358/Fuck_off_EA_App#faq) section in the readme, and my question was not answered there, or was not answered sufficiently
- [X] I have ran general troubleshooting steps (if applicable), such as restarting the PC/reinstalling Origin/whatever else is described in the [FAQ](https://github.com/p0358/Fuck_off_EA_App#faq), and they didn't help
- [X] I have checked [existing issues](https://github.com/p0358/Fuck_off_EA_App/issues?q=is%3Aissue) (including closed ones), and there wasn't an existing issue pertaining to my problem (with possible solutions in the comments)

### Your issue

It seems that after the Vol. 7 update, the game just won't boot anymore. I'm starting the game from Steam, Steam then opens Origin, but will instantly close again. Another game that I have tested, Battlefield 2042, boots up completely fine with Steam through Origin. 
It looks like in the Vol. 7 update for NfS Unbound, they have implemented a check if EA App is running on execution and if it's not, it's not even starting up. I haven't tested if it's working with EA App, so this is just a theory.

**Comment by LoVega1337** [comment]
I'm assuming #54 is related to this issue (even though I don't think Sims 3 received an update for quite a while).

**Comment by misterright1** [comment]
it's a shot in the dark but what if we had just an empty .exe file with the eaapp name to replicate it running.

**Comment by misterright1** [comment]
yeah.... no, I made a batch script to print out the number 1 every 10 thousand seconds and named it as EADesktopApp.exe and EALauncher.exe and no dice still the same.

**Comment by p0358** [comment]
> it's a shot in the dark but what if we had just an empty .exe file with the eaapp name to replicate it running.

If anything they'd check parent process, so you'd need to start up your game from within it as child. The problem is, the way Origin DRM works, the actual .exe only tells Origin to start up the game, then quits, then Origin starts up the game exe once again, having it attached as the parent.

So this workaround would only work if you used some kind of unpacker (there are ones out there) that removes Origin DRM from the game exe. Whether that's piracy depends on whether the game still later tries to connect to Origin LSX. For example with Titanfalls, it will generally still later connect to Origin, authenticate with it and work as if nothing happened, and a custom launcher fixes many adjacent issues. And without Origin account that owns the game, it'd be no dice, you could only watch the title screen. But how other games might have it implemented on the other hand, idk.

Also if the cause of the problems is client-side anti-cheat, then the matter and detection methods used might be trickier, and I'm not to say that tripping some checks couldn't result in some kind of ban either, so beware of that too I guess...

**Comment by Augusto2000** [comment]
I have the same problem, regardless of how I open it, it asks to install the EA App..

**Comment by misterright1** [comment]
yeah bit the bullet myself aswell installed the EA launcher, game wouldn't launch unless I had bitdefender off so I did indeed make an exception. If I have more time these days I might try to launch the game via the Origin app.

**Comment by misterright1** [comment]
Yeah a quick update, game is excluded in the antivirus and the antivirus is disabled, but with origin installed the game does not launch.
